src/workloads/execution/EventCollection.yml

```diff
@@ -0,0 +1,71 @@
+SchemaVersion: 2018-07-01
+Owner: Storage Execution
+Description: |
+  Inserts many documents into an event collection.
+
+GlobalDefaults:
+  Collection: &Collection Collection0
+  Database: &db test
+
+Actors:
+  - Name: CreateEventCollection
+    Type: RunCommand
+    Phases:
+      - Repeat: 1
+        Database: *db
+        Operation:
+          OperationName: RunCommand
+          OperationCommand:
+            drop: *Collection
+      - Repeat: 1
+        Database: *db
+        Operation:
+          OperationName: RunCommand
+          OperationCommand:
+            create: *Collection
+            clusteredIndex:
+              { key: { _id: 1 }, unique: true, autoAssign: "timestamp" }
+      - &Nop { Nop: true }
+  - Name: Insert
+    Type: Loader
+    Database: *db
+    Threads: 1
+    Phases:
+      - *Nop
+      - *Nop
+      - Repeat: 1
+        Database: *db
+        Threads: 1
```

Comment:
Does this mean all of the writes are happening on a single thread? We'll want the workload to test when there are multiple threads writing as well, since that's where any insert serialization we implement will be noticeable. In fact, maybe we want multiple loader phases each with a different level of concurrency? Maybe 1, 4, 8, 16?